server/master/src/main/java/org/apache/accumulo/master/tableOps/CreateDir.java

```diff
@@ -47,5 +58,23 @@ public void undo(long tid, Master master) throws Exception {
     VolumeManager fs = master.getFileSystem();
     fs.deleteRecursively(new Path(tableInfo.dir));
 
+    if (tableInfo.initialSplitSize > 0) {
+      SortedSet<Text> dirInfo = Utils
+          .getSortedSetFromFile(master.getInputStream(tableInfo.splitDirsFile), false);
+      for (Text dirname : dirInfo) {
+        fs.deleteRecursively(new Path(dirname.toString()));
+      }
+    }
+  }
+
+  private void createTabletDirectories(VolumeManager fs, SortedSet<Text> dirInfo)
+      throws IOException {
+
+    for (Text info : dirInfo) {
+      if (fs.exists(new Path(info.toString())))
```

Comment:
If the Repo fails and is later run again, would this cause it to fail?

Reply:
Thanks for catching that. There is no need to fail if the directory already exists upon retry.